### PR TITLE
Implement throughput measurement function

### PR DESCRIPTION
**Pull request by NeiroYT**
Solving #11

### CODE REVIEW

File: test/test.cpp [hunk 1]
```diff
@@ -332,3 +333,60 @@ TEST(accuracy, accuracy_norm_throws_when_bad_pointer) {
...
+    const std::vector<std::vector<T> > &first,
+    const std::vector<std::vector<T> > &second) {
+  std::vector<std::vector<T> > sum = first;
+  for (size_t i = 0; i < first.size(); i++) {
+    for (size_t j = 0; j < first.size(); j++) {
+      sum[i][j] = sum[i][j] + second[i][j];
+    }
+  }
+  return sum;
+}
```
Comment (aobolensk) [inline test/test.cpp:351]: Replace with std::accumulate

**Review by aobolensk — CHANGES_REQUESTED** [review]

File: test/test.cpp [hunk 2]
```diff
@@ -332,3 +333,105 @@ TEST(accuracy, accuracy_norm_throws_when_bad_pointer) {
...
+
+// ==========================
+// Throughput tests
+
+template <typename T>
+std::vector<T> matrix_sum(const std::vector<T> &first,
+                          const std::vector<T> &second) {
+  std::vector<T> res(first);
+  for (size_t i = 0; i < first.size(); i++) {
+    res[i] += second[i];
```
Comment (allnes) [inline test/test.cpp:345]: please use `std::accumulate`

**Review by allnes — CHANGES_REQUESTED** [review]

File: test/test.cpp (hunk 2 above)
Comment (NeiroYT) [reply test/test.cpp:345]: I thought `std::accumulate` calculates sum in range, not sum of vectors

Comment (allnes) [reply test/test.cpp:345]: sorry, my mistake - not `std::accumulate`, use `std::transform`

**Review by aobolensk — APPROVED** [review]

**Review by allnes — APPROVED** [review]

**Review by AndreySorokin7 — APPROVED** [review]

**Pull request merged**